### PR TITLE
Update the release version in badges and remove build badges

### PR DESCRIPTION
**Pull request by schoims**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -25,21 +25,13 @@ To prepare for this change, make sure your device's certificate store has both o
...
-| [Preview](https://github.com/Azure/azure-iot-sdk-csharp/tree/preview) | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/CSharp%20Canary%20-%20Central%20US%20EUAP?branchName=preview)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build/latest?definitionId=402&repositoryFilter=9&branchName=preview) |
-
 ## Recommended NuGet packages
 
 | Package Name                                          | Release Version                                           |
 | ---                                                   | ---                                                       |
 | Microsoft.Azure.Devices.Client                        | [![NuGet][iothub-device-release]][iothub-device-nuget]    |
 | Microsoft.Azure.Devices                               | [![NuGet][iothub-service-release]][iothub-service-nuget]  |
+| Microsoft.Azure.Devices.Authentication                | [![NuGet][iothub-auth-release]][iothub-auth-nuget]        |
 | Microsoft.Azure.Devices.Shared                        | [![NuGet][iothub-shared-release]][iothub-shared-nuget]    |
```
Comment (brycewang-microsoft) [inline readme.md:35]: These nugets are deprecated and for v1 only, I think we need to remove them from README. Agree? @abhipsaMisra

Comment (abhipsaMisra) [inline readme.md:35]: you'll also want to remove the shared nuget and provisioning mqtt/amqp/http nugets from this list, and add in the authentication provider nuget

Comment (abhipsaMisra) [reply readme.md:35]: That's correct, yes.

File: SDK v2 migration guide.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ are outlines of the notable breaking changes as well as a mapping from version 1
    - [IoT hub device client](#iot-hub-device-client)
    - [IoT hub service client](#iot-hub-service-client)
    - [Device Provisioning Service (DPS) device client](#dps-device-client)
-   - [DPS service slient](#dps-service-client)
-   - [Security provider clients](#security-provider-clients)
```
Comment (drwill-ms) [inline SDK v2 migration guide.md:16]: We use the old terms and point them to the new ones, so this old line was correct.

File: readme.md [hunk 3]
```diff
@@ -25,26 +25,14 @@ To prepare for this change, make sure your device's certificate store has both o
...
-| [Preview](https://github.com/Azure/azure-iot-sdk-csharp/tree/preview) | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/CSharp%20Canary%20-%20Central%20US%20EUAP?branchName=preview)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build/latest?definitionId=402&repositoryFilter=9&branchName=preview) |
-
 ## Recommended NuGet packages
 
 | Package Name                                          | Release Version                                           |
 | ---                                                   | ---                                                       |
 | Microsoft.Azure.Devices.Client                        | [![NuGet][iothub-device-release]][iothub-device-nuget]    |
 | Microsoft.Azure.Devices                               | [![NuGet][iothub-service-release]][iothub-service-nuget]  |
-| Microsoft.Azure.Devices.Shared                        | [![NuGet][iothub-shared-release]][iothub-shared-nuget]    |
+| Microsoft.Azure.Devices.Authentication                | [![NuGet][iothub-auth-release]][iothub-auth-nuget]        |
```
Comment (drwill-ms) [inline readme.md:34]: This change is good, though.

File: readme.md [hunk 4]
```diff
@@ -25,26 +25,14 @@ To prepare for this change, make sure your device's certificate store has both o
...
 | Microsoft.Azure.Devices.Provisioning.Client           | [![NuGet][dps-device-release]][dps-device-nuget]          |
-| Microsoft.Azure.Devices.Provisioning.Transport.Amqp   | [![NuGet][dps-device-amqp-release]][dps-device-amqp-nuget]|
-| Microsoft.Azure.Devices.Provisioning.Transport.Http   | [![NuGet][dps-device-http-release]][dps-device-http-nuget]|
-| Microsoft.Azure.Devices.Provisioning.Transport.Mqtt   | [![NuGet][dps-device-mqtt-release]][dps-device-mqtt-nuget]|
 | Microsoft.Azure.Devices.Provisioning.Service          | [![NuGet][dps-service-release]][dps-service-nuget]        |
 | Microsoft.Azure.Devices.Provisioning.Security.Tpm     | [![NuGet][dps-tpm-release]][dps-tpm-nuget]                |
```
Comment (drwill-ms) [inline readme.md:37]: This TPM one can be removed.

File: readme.md [hunk 5]
```diff
@@ -232,21 +220,15 @@ To learn more, review the [privacy statement](https://go.microsoft.com/fwlink/?L
...
-[dps-device-amqp-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Transport.Amqp/
-[dps-device-http-release]: https://img.shields.io/nuget/v/Microsoft.Azure.Devices.Provisioning.Transport.Http.svg?style=plastic
-[dps-device-http-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Transport.Http/
-[dps-device-mqtt-release]: https://img.shields.io/nuget/v/Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.svg?style=plastic
-[dps-device-mqtt-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Transport.Mqtt/
-[dps-service-release]: https://img.shields.io/nuget/v/Microsoft.Azure.Devices.Provisioning.Service.svg?style=plastic
+[dps-service-release]: https://img.shields.io/nuget/vpre/Microsoft.Azure.Devices.Provisioning.Service?style=plastic
 [dps-service-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Service/
 [dps-tpm-release]: https://img.shields.io/nuget/v/Microsoft.Azure.Devices.Provisioning.Security.Tpm.svg?style=plastic
 [dps-tpm-nuget]: https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Security.Tpm/
```
Comment (drwill-ms) [inline readme.md:234]: remove

**Review by drwill-ms — APPROVED** [review]
Comments inline

File: SDK v2 migration guide.md (hunk 2 above)
Comment (schoims) [reply SDK v2 migration guide.md:16]: Sure, I will make sure to update the broken link only

**Pull request merged**